aiplatform/pom.xml

```diff
@@ -89,6 +105,11 @@
       <version>1.7.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-testing</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Please remove this dependency and use the library directly with mockito to mock the test.